client/container_logs.go

```diff
@@ -22,6 +23,12 @@ type ContainerLogsOptions struct {
 	Details    bool
 }
 
+// ContainerLogsResult is the result of a container logs operation.
+type ContainerLogsResult struct {
+	rc    io.ReadCloser
+	close func() error
+}
```

Comment:
This is one we need to improve, so that consumers don't have to bother with the `stdcopy` package; we may not be able to do that for this release, but definitely should before v1.0 of the client.